api/authorization-api-1_0.md

```diff
@@ -67,187 +67,292 @@ normative:
+The Authorization API is a transport-agnostic API published by the PDP, to which the PEP acts as a client. The Transport section of this specification details possible bindings, such as HTTPS or gRPC.
 
-Authorization for the Authorization API itself is out of scope for this document, since authorization for REST APIs is well-documented elsewhere. For example, the Authorization API MAY support authorization using an `Authorization` header, using a `basic` or `bearer` token. Support for OAuth 2.0 ({{RFC6749}}) is RECOMMENDED. 
+Authorization for the Authorization API itself is out of scope for this document, since authorization for APIs is well-documented elsewhere. For example, the Authorization API's HTTPS binding MAY support authorization using an `Authorization` header, using a `basic` or `bearer` token. Support for OAuth 2.0 ({{RFC6749}}) is RECOMMENDED. 
 
 ## Features
-The Authorization API has two main features:
-
-* An Access Evaluation API, which enables a PEP to find out if a specific request can be permitted to access specific resources. The following are non-normative examples:
-  * Can Alice view document #123?
-  * Can a manager print?
-* A Search API, which enables a PEP to ask open-ended questions. This feature is sometimes referred to as partial evaluation, reverse querying, or contextual access queries by other frameworks. The following are non-normative examples:
-  * What can happen?
-  * What can Alice do?
-  * Who can access record #123?
-  * Who can edit medical records between 8 am and 3 pm?
-  * Who can edit?
-  * What can happen at 3 pm from Glasgow?
-
-# API Specification
-The Authorization API has two parts, Access Evaluation and Search. Each of these is defined below. The definition is transport-agnostic. Refer to the Transport section for example bindings.
+The core feature of the Authorization API is the Access Evaluation API, which enables a PEP to find out if a specific request can be permitted to access a specific resource. The following are non-normative examples:
+
+- Can Alice view document #123?
+- Can Alice view document #123 at 16:30 on Tuesday, June 11, 2024?
+- Can a manager print?
 
 ## API Version
 This document describes the API version 1. Any updates to this API through subsequent revisions of this document or other documents MAY augment this API, but MUST NOT modify the API described here. Augmentation MAY include additional API methods or additional parameters to existing API methods, additional authorization mechanisms, or additional optional headers in API requests. All API methods for version 1 MUST be immediately preceded by the relative URL path `/v1/`.
 
 ## Information Model
+The information model for requests and responses include the following entities: Subject, Action, Resource, Context, and Decision. These are all defined below.
```

Comment:
So, I'm now having second thoughts about allowing additional categories... For instance, what if I want to ask: "Can Alice send $100 to Bob?". Historically I modeled this as Alice in Subject, send in Action and the rest (amount and recipient) in resource. But... we could also introduce a recipient-subject category. Too much?

Reply:
I don't see a reason to revisit SARE / PARC...

There are a few ways of breaking up the action and resource, but I think the most natural way to do it is that the Action is "Send" and the Resource is of type:user, id:Bob, amount:$100.

Reply:
resolved during discussion - keeping SARC model